Makefile

```diff
@@ -48,11 +48,13 @@ run-dev:
 
 build:
 	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/gravity cmd/gravity/single_binary.go
+	$(GOBUILD) -ldflags '$(LDFLAGS)' -race -o bin/gravity-race cmd/gravity/single_binary.go
```

Comment:
where would we use this one?

Reply:
Use it anywhere when it is hard to detect race in test/dev. I will use it on production to re-product a race bug.